msteams-platform/bots/what-are-bots.md

```diff
@@ -49,6 +49,8 @@ With Microsoft Graph APIs for calls and online meetings, Teams apps can now inte
 
 You can use the Teams bot APIs to get information for members of a chat or team. See [changes to Teams bot APIs for fetching team or chat members](~/resources/team-chat-member-api-changes.md).
 
+Embedding an existing bot as a Teams bot isn't possible. You can create a new Teams bot by using the back-end infrastructure and your business logic.
```

Comment:
@v-npaladugu , Wajeed and I discussed more than what's reflected in this line. I recommend taking notes please.

1. Mention first that for security reasons, apps do not allow as is embedding of any existing bot. 
1. This means that dev have to create a new bot for their Teams app.
1. However, to make their life easier, they can reuse the business logic + backend infrastructure in the Teams bot.

Reply:
An update @v-npaladugu , do not mention anything about this particular issue. Also, remove the edit `Embedding an existing bot as a Teams bot isn't possible. You can create a new Teams bot by using the back-end infrastructure and your business logic.` I've updated our shared spreadsheet as well.